rskj-core/src/test/java/co/rsk/peg/ErpFederationTest.java

```diff
@@ -0,0 +1,98 @@
+package co.rsk.peg;
+
+import co.rsk.bitcoinj.core.BtcECKey;
+import co.rsk.bitcoinj.core.NetworkParameters;
+import co.rsk.bitcoinj.script.Script;
+import java.time.ZonedDateTime;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.bouncycastle.util.encoders.Hex;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ErpFederationTest {
+    private ErpFederation federation;
+
+    // ERP federation keys
+    List<BtcECKey> erpFedKeys = Arrays.stream(new String[]{
```

Comment:
```suggestion
    private static List<BtcECKey> ERP_FED_KEYS = Arrays.stream(new String[]{
```
It's a constant value, right?